src/main/java/me/ryanhamshire/GriefPrevention/EntityEventHandler.java

```diff
@@ -106,8 +106,8 @@
 public class EntityEventHandler implements Listener
 {
     //convenience reference for the singleton datastore
-    private DataStore dataStore;
-    GriefPrevention instance;
+    private final DataStore dataStore;
+    private final GriefPrevention instance;
```

Comment:
I probably shouldn't have included an access change in this PR, but I really can't see this affecting addons - the only way to obtain the EntityEventHandler instance would be to iterate over registered listeners and grab it. Everyone sane would just use GriefPrevention.instance instead.